Dockerfile.dev

```diff
@@ -14,10 +14,9 @@ COPY scripts/execute-cleanup.sh .
 COPY setup.cfg .
 COPY scripts/service_env.sh scripts/
 
-EXPOSE 9006
+RUN chmod 777 -R .
 
-ENV FLASK_APP ./src/mediaserver.py
-ENV FLASK_ENV development
+EXPOSE 9006
 
 ENTRYPOINT ["./entrypoint.sh"]
-CMD ["flask", "run", "--host", "0.0.0.0", "--port", "9006"]
+CMD exec flask --app src/mediaserver run --host 0.0.0.0 --port 9006 --debug
```

Comment:
Why `--debug`?

Reply:
To enable the automatic reload of flask in test mode